src/Components/GrotError.tsx

```diff
@@ -35,13 +34,14 @@ export const GrotError = ({ children }: React.PropsWithChildren<Props>) => {
   return (
     <div className={styles.wrap}>
       <div className={styles.graphicContainer}>
-        <SVG
```

Comment:
I spent a couple of hours trying to solve this error. I suppose it has to do with incompatible `@types/react`. Using img seems to work fine here though. It would be good for someone to verify this, the styles may need some modifications.

Reply:
I did run into this error in another PR and magically yarn installing again seemed to fix it https://github.com/grafana/logs-drilldown/pull/1162/files/b63db2736101dc36848756745c222bdf9ddd2763#diff-c56d554d50f3861240e8830d52e68fc8631e930ce24f3b15dec9d80ed60a4e56. I'm thinking of merging this pr [1162](https://github.com/grafana/logs-drilldown/pull/1162/files/b63db2736101dc36848756745c222bdf9ddd2763#diff-c56d554d50f3861240e8830d52e68fc8631e930ce24f3b15dec9d80ed60a4e56) first and then updating this one with the changes.

Reply:
Hopefully that will fix the unit test failures on this branch as well 🤞

Reply:
The <SVG> is back now